fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -155,6 +207,12 @@ class MirabufSceneObject extends SceneObject {
 
         return mesh
     }
+
+    private EnablePhysics() {
+        this._mirabufInstance.parser.rigidNodes.forEach(rn => {
+            World.PhysicsSystem.EnablePhysicsForBody(this._mechanism.GetBodyByNodeId(rn.id)!)
+        })
+    }
```

Comment:
I added that to solve a bug so that is a good point and ill add that. The problem is that all of that stuff is from PR #1009 so I might need to wait for that to be merged